__fixtures__/preflight/tailwind.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  corePlugins: {
+    preflight: false,
```

Comment:
Just curious will this mean by default it will be `false` and will that not impact existing user that uses the `globalStyles` that works for them?

Reply:
This is a good point. TailwindCSS enables the preflight by default, so this should probably default to true so that it has to be explicitly disabled

Reply:
`preflight: true` is the default - you've added a comment to a fixture test here.